feeds/wlan-ap/opensync/patches/19-events-protobuf-definitions.patch

```diff
@@ -106,7 +104,7 @@ Index: opensync-2.0.5.0/interfaces/opensync_stats.proto
 +        optional bool                   using11k                            = 9;
 +        optional bool                   using11r                            = 10;
 +        optional bool                   using11v                            = 11;
-+        optional uint32                 timestamp_ms                        = 12;
++        required uint64                 timestamp_ms                        = 12;
```

Comment:
I recall the suggestion that the timestamp_ms should be a required field came from the Cloud side.  Is the change coordinated with them?